grub2-efi/install

```diff
@@ -11,17 +11,23 @@ case "$target" in
   i?86 ) target=i386 ;;
   x86_64 | amd64 ) target=x86_64 ;;
   aarch64 ) target=arm64 ;;
-  armv7l ) target=arm ;;
+  arm* ) target=arm ;;
 esac
 
 target="$target-efi"
 
 echo "target = $target"
 
+# Systems without efivars should just fall back to install grub as removable
```

Comment:
please add some clarification why it helps, not just what you do. I see that no-vram do not update `boot-device'/`Boot*'  NVRAM  variables which looks ok as there is no efi vars, but how can removable help? or also link with explanation is enough as yast2-bootloader probably have to do same thing, so we need to know why.